app/home/controller.ts

```diff
@@ -16,12 +16,27 @@ export default class Home extends Controller.extend({
             return;
         }
 
-        yield model.save();
+        try {
+            yield model.save();
+        } catch (e) {
+            // Handle email already exists error
+            if (+e.errors[0].status === 400) {
+                const { options } = model.get('validations')._validators.email1
+                    .find(validator => validator._type === 'exclusion');
```

Comment:
If possible, I think a [computed option](http://offirgolan.github.io/ember-cp-validations/docs/modules/Advanced%20Usage.html#computedOptions) would be better here.

Reply:
Thanks for pointing that out. Didn’t realize it was an option. I’ll try it.